transformer_engine/paddle/constants.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+"""Constants"""
+import paddle
+import transformer_engine_paddle as tex
+"""
+Map from paddle dtype to TE dtype
+"""
+TE_DType = {
```

Comment:
```suggestion
# See LICENSE for license information.
"""Constants"""

import paddle
import transformer_engine_paddle as tex

"""
Map from paddle dtype to TE dtype
"""
TE_DType = {
```

Reply:
Done, but yapf wants to remove the new line at L16.